integration-tests/pom.xml

```diff
@@ -138,6 +143,14 @@
             <keycloak.realm.url></keycloak.realm.url>
 
             <webhook.site.uuid>${env.WEBHOOK_SITE_UUID}</webhook.site.uuid>
+
+            <!-- Managed Kafka properties -->
+            <event-bridge.rhoas.sso.mas.auth-server-url>${keycloak.realm.url}</event-bridge.rhoas.sso.mas.auth-server-url>
+            <event-bridge.kafka.bootstrap.servers>${env.MANAGED_KAFKA_BOOTSTRAP_SERVER}</event-bridge.kafka.bootstrap.servers>
+            <event-bridge.rhoas.sso.mas.client-id>${env.MANAGED_KAFKA_ADMIN_CLIENT_ID}</event-bridge.rhoas.sso.mas.client-id>
+            <event-bridge.rhoas.sso.mas.client-secret>${env.MANAGED_KAFKA_ADMIN_CLIENT_SECRET}</event-bridge.rhoas.sso.mas.client-secret>
+            <event-bridge.kafka.client.id>${env.MANAGED_KAFKA_OPS_CLIENT_ID}</event-bridge.kafka.client.id>
+            <event-bridge.kafka.client.secret>${env.MANAGED_KAFKA_OPS_CLIENT_SECRET}</event-bridge.kafka.client.secret>
```

Comment:
Is there any reason to use such property names? It doesn't have any relation to the manager itself, right?

Reply:
I used these names because it corresponds to the following run scripts for manager: https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/dev/bin/manager-run.sh
Did not want to think up any new names.

Reply:
I see.
Though it is not much clear from user of the tests....
for example, `event-bridge.rhoas.sso.mas.client-id` - that is it used in tests? without code inspection the property name doesn't give me any idea.